config/locales/en.yml

```diff
@@ -45,7 +45,7 @@ en:
   minutes:
     one: a minute
     other: '%{count} minutes'
-  paste_title_html: "%{title} %{private} created by %{author} will expire in %{expires_in}"
+  paste_title_html: "%{title} %{private} %{marked} created by %{author} will expire in %{expires_in}"
```

Comment:
If I understand correctly, this will either yield "unclassified" or "spam". I think marking spam is a good idea, but not giving any mark for non-spam might be more easy to understand than "unclassified"?

Reply:
There are 3 classifications, unclassified, ham and spam

Reply:
Right, I just figured this would look a bit odd in public /pastes:

```
mypaste unclassified/ham created by Victorious Squirrel will expire in 7 days 
```

But maybe if it is stylized similarly as the "private" label is it is fine.

Reply:
It looks exactly the same

Reply:
In blue?
My point is, the average user knows what "spam" means, but probably not "ham" and "unclassified" (at least not in this context).

Reply:
Thanks. :)